manifests/base/argo-server/argo-server-deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: argo-server
+spec:
+  selector:
+    matchLabels:
+      app: argo-server
+  template:
+    metadata:
+      labels:
+        app: argo-server
+    spec:
+      serviceAccountName: argo-server
+      containers:
+        - name: argo-server
+          image: argoproj/argo-server:latest
+          args: [--auth-mode, client]
+          ports:
+            - containerPort: 2746
+          readinessProbe:
+            httpGet:
+              port: 2746
+              scheme: HTTP
+              path: /
+            initialDelaySeconds: 5
+            periodSeconds: 30
+          env:
+            # TODO - maybe unused
+            - name: ARGO_NAMESPACE
```

Comment:
Yes I think this was only for the old, node-based API server and can be removed.

Reply:
done